packages/env/lib/cli.js

```diff
@@ -109,6 +109,28 @@ module.exports = function cli() {
 		},
 		withSpinner( env.clean )
 	);
+	yargs.command(
+		'log',
```

Comment:
Could we make it `logs`? That way it matches `docker-compose` (and my muscle memory 😂)

Reply:
done!